run-build.ps1

```diff
@@ -30,6 +30,7 @@ if($Help)
 $env:CONFIGURATION = $Configuration;
```

Comment:
There's currently no bootstrap.sh, I'll see if there's an existing issue on that.